lua/neorepl/init.lua

```diff
@@ -121,23 +121,22 @@ package.loaded['neorepl.bufs'] = bufs
 
 local neorepl = {}
 
+local current_config = vim.deepcopy(DEFAULT_CONFIG)
+
 ---Set default configuration
 ---@param config? neorepl.Config
 function neorepl.config(config)
-  -- TODO: merge with the old one.
-  -- to do so, we need a way of resetting values back to default,
-  -- in particular on_init, env_lua, lang. either false or have neorepl.DEFAULT constant
-  config = validate(config)
+  config = validate(config or {})
```

Comment:
`validate` already does `or {}` internally? Am I missing something?

Reply:
Ah yes, didn't see it, will change that